lib/runner.rb

```diff
@@ -0,0 +1,13 @@
+require './lib/stat_tracker'
+
+game_path = './data/games.csv'
+team_path = './data/teams.csv'
+game_teams_path = './data/game_teams.csv'
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
```

Comment:
This'll be interesting... I'm looking forward to seeing all of our ideas on how to access the 'locations' hash